packages/firebase_auth/firebase_auth_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.2
+
+* Implement `fetchSignInMethodsForEmail`, `isSignInWithEmailLink`, and `sendLinkToEmail`. 
```

Comment:
Can you add some unit tests? At least, it'd be interesting to have a test of the sad path of `signInWithEmailLink`, that codifies whatever is decided in this PR (whether it throws or catch + return null... whatever)

Reply:
Do we need to do that? I believe the `firebase-dart` package will properly throw the expected `FirebaseError` instances which #1698 will map to our expected `PlatformException`s.